app/middleware/parameter-validation.ts

```diff
@@ -27,6 +30,22 @@ function validateLinkTypeParameter(req: HarmonyRequest): void {
   }
 }
 
+/**
+ * Validate that the req query parameter names are correct according to Harmony implementation of OGC spec.
+ *
+ * @param req - The client request
+ */
+function validateCoverageRangesetParameterNames(req: HarmonyRequest): void {
+  const requestedParams = Object.keys(req.query);
+  const allowedParams = req.method == 'GET' ? coverageRangesetGetParams : coverageRangesetPostParams;
```

Comment:
will `method` always be uppercase? Should this be a case-insensitive string compare?